src/components/common/icons/token_icon.tsx

```diff
@@ -69,9 +70,39 @@ export const TokenIcon = (props: Props) => {
     const { token } = props;
     const TokenIconComponentName = getTokenIconNameBySymbol(token.symbol) as keyof typeof TokenIcons;
     const Icon: React.FunctionComponent = TokenIcons[TokenIconComponentName];
-    return Icon ? React.createElement(Icon, null, null) : null;
+    return Icon
+        ? React.createElement(Icon, null, null)
+        : fallbackIcon({
+              fill: (token && token.primaryColor) || randomColor(),
```

Comment:
@mariano-aguero : can we get rid of the `randomColor()` ? I may become a little odd to the user if the color of the icon for a token is different every time the browser is refreshed.

Also `primaryColor` is now part of the corresponding TS interface: I imagine this will encourage the addition of Tokens' Metadata with that info.

Worst case scenario, use a default constant and boring-grey color.

Reply:
> I may become a little odd to the user if the color of the icon for a token is different every time the browser is refreshed.

Is that so? I mean, a few apps and sites do that when they don't have a stablished color / icon / image for some elements.

This Whatsapp's user names, for example. Or Google's apps and its avatars.

Reply:
Done